styles.css

```diff
@@ -0,0 +1,142 @@
+/*body{
+  height: 100%;
+  flex-direction: column;
+}*/
+
+.main{
+  height: 75%;
+}
+.footer{
+  height: 25%;
+}
+
+.top{
+  display: flex;
+  background-color: #F5C85F;
+  height: 10px;
+  width: auto;
+}
+
+.images {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  align-content: center;
+  padding-top: 40px;
+  padding-bottom: 40px;
```

Comment:
You can group this two in: `padding: 40px 0;`. The first value is for top-bottom and the second is for left-right